proc_spincore_IR.py

```diff
@@ -0,0 +1,225 @@
+from pyspecdata import *
```

Comment:
two problems here -- this seems to be added to the root directory.

It's also implying there are different files to process the inversion recovery for the spincore vs. bruker, which is not what we want.

Reply:
you want one file that you test on both bruker and spincore data

Reply:
done

Reply:
so what is supposed to process the spincore data now?  I would have guessed `proc_IR.py`, but it doesn't seem to have any flexibility in terms of which phase cycling dimensions are present.

Reply:
proc_IR.py does both the main reason for the two files was the preprocessing was different and loading in the files was different (back when I first started this adventure and made a million scripts instead of compiling). shouldn't the coherence pathway be the same for any IR experiment? I'm a bit confused on your question. As far as I'm aware they should be processed the same (slice the freq range, apply phase corrections/clock correction if needed, align data, integrate and fit)....am I missing something that is different between the two?

Reply:
OK -- could be I'm mixing things up, then.  I thought that on the bruker you were doing more or less phase cycling than on the spincore.  For the coherence selection to be the same, the phase cycling needs to be the same.